src/rust/demikernel/libos/mod.rs

```diff
@@ -137,6 +152,17 @@ impl LibOS {
         }
     }
 
+    /// Opens an existing memory queue and connects to consumer end.
+    pub fn open_pipe_for_pop(&mut self, name: &str) -> Result<QDesc, Fail> {
```

Comment:
Since this is just a temporary solution, shouldn't we annotate it as deprecated?